operator/pkg/model/translation/envoy_listener_test.go

```diff
@@ -113,4 +126,18 @@ func TestNewSNIListener(t *testing.T) {
 		require.Len(t, listener.GetFilterChains(), 1)
 		require.Len(t, listener.GetFilterChains()[0].FilterChainMatch.ServerNames, 2)
 	})
-}
+
+	t.Run("normal SNI listener with Proxy Protocol", func(t *testing.T) {
+		res, err := NewSNIListener("dummy-name", map[string][]string{"dummy-namespace/dummy-service:443": {"example.org", "example.com"}}, WithProxyProtocol())
+		require.Nil(t, err)
+
+		listener := &envoy_config_listener.Listener{}
+		err = proto.Unmarshal(res.Value, listener)
+		require.Nil(t, err)
+
+		require.Equal(t, "dummy-name", listener.Name)
+		require.Len(t, listener.GetListenerFilters(), 2)
+		require.Len(t, listener.GetFilterChains(), 1)
+		require.Len(t, listener.GetFilterChains()[0].FilterChainMatch.ServerNames, 2)
+	})
+}
```

Comment:
[Travis-CI build failed](https://app.travis-ci.com/github/cilium/cilium/builds/266440058) with:

```
Unformatted Go source code:

./operator/pkg/model/translation/envoy_listener_test.go

diff ./operator/pkg/model/translation/envoy_listener_test.go.orig ./operator/pkg/model/translation/envoy_listener_test.go

--- ./operator/pkg/model/translation/envoy_listener_test.go.orig

+++ ./operator/pkg/model/translation/envoy_listener_test.go

@@ -140,4 +140,4 @@

 		require.Len(t, listener.GetFilterChains(), 1)

 		require.Len(t, listener.GetFilterChains()[0].FilterChainMatch.ServerNames, 2)

 	})

-}

\ No newline at end of file

+}
```

please add an EOL at the end of the file.

Reply:
@kaworu done